.guides/setup.md

```diff
@@ -0,0 +1,59 @@
+# Cloud Functions for Firebase setup guide
+
+This guide provides a step-by-step process for setting up a new Firebase Functions project, tailored for a coding agent.
+
+## 1. Create a 2nd-gen HTTP function
+
+Replace the contents of `src/index.ts` (or `index.js`) with the following code to create a simple, modern v2 HTTP endpoint.
+
+```typescript
+import { onRequest } from "firebase-functions/https";
+import * as logger from "firebase-functions/logger";
+import { defineString } from "firebase-functions/params";
+
+// Define a configuration parameter for a greeting
+const greeting = defineString("GREETING", { default: "Hello" });
+
+export const helloWorld = onRequest({ maxInstances: 1 }, async (request, response) => {
+  logger.info("Request received!", { structuredData: true });
+  response.send(`${greeting.value()} from Firebase!`);
+});
+```
```

Comment:
I know token is scarce, but wondering if we have any wiggle room to add an event triggered function example (firestore) that show use of params INSIDE the function parameters (e.g. min instance)

Reply:
`setup.md` is a directly invoked prompt vs. something included in system instructions, so no token constraints here. Go nuts!

Reply:
added